Modul/models.py

```diff
@@ -0,0 +1,95 @@
+import random
+import settings
+
+
+initial_player_health = settings.INITIAL_PLAYER_HEALTH
```

Comment:
```suggestion
from settings import INITIAL_PLAYER_HEALTH as initial_player_health
```
Хоча я не зовсім розумію, чому б не скористатись просто `settings.INITIAL_PLAYER_HEALTH`.
Тобі релігія забороняє чи що?

Reply:
)))